lib/worker.js

```diff
@@ -325,6 +333,25 @@ function _gossipWith(ledgerNode, voter, blockHeight, peer, callback) {
   });
 }
 
+function _syncVotes(options, callback) {
+  async.each(options.election, (e, callback) => {
+    async.auto({
+      hashes: callback => async.map(e.electionResults, api._hasher, callback),
```

Comment:
We want to avoid recomputing all these hashes (it is very expensive). We already have to do it when calling `certify`. If we don't do `syncVotes` and `certify` in parallel above, perhaps we can figure out a avoid it.